src/API/Site/Controllers/MerchantCenter/ContactInformationController.php

```diff
@@ -124,35 +124,40 @@ protected function get_contact_information_endpoint_edit_callback(): callable {
 	 */
 	protected function get_schema_properties(): array {
 		return [
-			'id'                      => [
+			'id'                        => [
 				'type'              => 'integer',
 				'description'       => __( 'The Merchant Center account ID.', 'google-listings-and-ads' ),
 				'context'           => [ 'view', 'edit' ],
 				'validate_callback' => 'rest_validate_request_arg',
 			],
-			'phone_number'            => [
+			'phone_number'              => [
 				'type'        => 'string',
 				'description' => __( 'The phone number associated with the Merchant Center account.', 'google-listings-and-ads' ),
 				'context'     => [ 'view' ],
 			],
-			'mc_address'              => [
+			'phone_verification_status' => [
+				'type'        => 'string',
```

Comment:
❓  I guess this is an enum 'verified'|'unverified' ?? Or it could be more things?

Reply:
Added in https://github.com/woocommerce/google-listings-and-ads/pull/1732/commits/bba9fa36cfd892cc58e47309521fe49b466cf2e0.